### PR TITLE
Update Bootstrap: Elements Details root folder

### PR DESCRIPTION
**Pull request by maiwald**

### CODE REVIEW

File: app/packs/src/apps/mydb/elements/details/ElementDetailSortTab.js
Comment (beque) [inline app/packs/src/apps/mydb/elements/details/ElementDetailSortTab.js:1]: Have you tested the resize functionality?

File: app/packs/src/apps/mydb/elements/details/NumericInputUnit.js [hunk 1]
```diff
@@ -103,35 +103,32 @@ export default function NumericInputUnit(props) {
...
     <div className={`numericInputWithUnit_${unit}`}>
-      {labelWrap}
+      {label
+        ? <Form.Label>{label}</Form.Label>
+        : <Form.Label style={{ paddingTop: '15px' }} />}
       <InputGroup>
         <Form.Control
           type="text"
-          bsClass="bs-form--compact form-control"
+          className="bs-form--compact"
```
Comment (beque) [inline app/packs/src/apps/mydb/elements/details/NumericInputUnit.js:116]: You could replace class "bs-form-compact" with "px-1"
This class has just this settings:
padding-left: 5px !important;
padding-right: 5px !important;

But when you change this the cypress test will fail
/spec/cypress/end_to_end/manage_samples.cy.js:
This test needs a specific class name

Maybe you can add a className for the test?
Or do you have an other idea?

File: app/packs/src/apps/mydb/elements/details/PrivateNoteElement.js [hunk 2]
```diff
@@ -85,39 +81,36 @@ export default class PrivateNoteElement extends React.Component {
 
     return (
       <div style={{ marginBottom: '10px' }}>
```
Comment (beque) [inline app/packs/src/apps/mydb/elements/details/PrivateNoteElement.js:83]: You can replace the style with className="mt-2"

**Review by beque — COMMENTED** [review]
Just some minor fixes

File: app/packs/src/apps/mydb/elements/details/NumericInputUnit.js [hunk 3]
```diff
@@ -103,35 +103,32 @@ export default function NumericInputUnit(props) {
...
-    </Button>
-  );
+  const bsSize = field === 'flash_point' ? 'sm' : null;
 
   return (
     <div className={`numericInputWithUnit_${unit}`}>
-      {labelWrap}
+      {label
+        ? <Form.Label>{label}</Form.Label>
+        : <Form.Label style={{ paddingTop: '15px' }} />}
```
[the remark this answers is not in the document]
Comment (maiwald) [reply app/packs/src/apps/mydb/elements/details/NumericInputUnit.js:112]: I'll go with `pt-2` for now. `ChemicalTab` is the only component that uses this one without a label. And `ChemicalTab` is used in `SampleDetails` and currently not visible. Joachim is working on that part and I don't want to change anything there while he is still on it.

File: app/packs/src/apps/mydb/elements/details/ElementDetailSortTab.js
Comment (maiwald) [reply app/packs/src/apps/mydb/elements/details/ElementDetailSortTab.js:1]: Yes. The popoup stays anchored to the button without rendering on window resize.

File: app/packs/src/apps/mydb/elements/details/LiteratureDetails.js [hunk 4]
```diff
@@ -30,33 +32,11 @@ import LiteraturesFetcher from 'src/fetchers/LiteraturesFetcher';
...
-CloseBtn.propTypes = {
-  onClose: PropTypes.func.isRequired,
-};
-
-const clipboardTooltip = () => (
-  <Tooltip id="assign_button">copy to clipboard</Tooltip>
-);
-
-const ElementLink = ({ literature }) => {
+function ElementLink({ literature }) {
```
[the remark this answers is not in the document]
Comment (maiwald) [reply app/packs/src/apps/mydb/elements/details/LiteratureDetails.js:39]: This comes from an es lint remark I keep seeing in my Editor:
https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/function-component-definition.md

I'll change it back. Maybe we can add our preferred style to the `.eslintrc`?

File: app/packs/src/apps/mydb/elements/details/LiteratureDetails.js [hunk 5]
```diff
@@ -361,127 +376,73 @@ export default class LiteratureDetails extends Component {
...
-                        <Button size="sm" active className="clipboardBtn" data-clipboard-text={contentElements} >
-                          <i className="fa fa-clipboard" />
-                        </Button>
-                      </OverlayTrigger>
-                    </Panel.Title>
-                  </Col>
-                </Row>
-              </Panel.Heading>
-              <Panel.Body collapsible="true">
+      <Card className="format-analysis-panel">
```
[the remark this answers is not in the document]
Comment (maiwald) [reply app/packs/src/apps/mydb/elements/details/LiteratureDetails.js:379]: I'll remove the class from the `Card`. The class is still used in `src/apps/mydb/elements/details/formats/FormatComponent.js` but I can pick that next and remove the class in that PR.

File: app/packs/src/apps/mydb/elements/details/LiteratureDetails.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (beque) [reply app/packs/src/apps/mydb/elements/details/LiteratureDetails.js]: Ah OK.
eslint does not work in my editor.
I have just js prettier
If you can add this to eslintrc would maybe a question for Pierre

**Pull request merged**